x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -216,6 +217,8 @@ export function WorkspacePanel({
     }
   }
 
+  const value = useMemo(() => ({ id: 'lnsWorkspace' }), []);
```

Comment:
nit: As this is just a constant, you can create it once in the module scope - no need for a `useMemo`